types/black-engine/Engine.d.ts

```diff
@@ -47,9 +47,9 @@ export class Engine extends MessageDispatcher {
     private __bootStage;
     private __onVisibilityChangeFallback;
     __onVisibilityChange(): void;
-    hasSystem(systemTypeName: () => void): boolean;
+    hasSystem(systemTypeName: any): boolean;
     private __addSystem;
-    getSystem(typeName: () => void): System | null;
+    getSystem<T>(typeName: T): T | null;
```

Comment:
It seems strange to me that the `typeName` and the `System` would have the same type.